src/extensibility/node/ExtensionManagerDomain.js

```diff
@@ -207,7 +207,7 @@ function legacyPackageCheck(legacyDirectory) {
  * @param {function} callback (err, result)
  * @param {boolean} _doUpdate  private argument to signal that an update should be performed
  */
-function _cmdInstall(packagePath, destinationDirectory, options, callback, _doUpdate) {
+function _cmdInstall(packagePath, destinationDirectory, options, callback, pCallback, _doUpdate) {
```

Comment:
there's a reason, progress callback was added right after callback argument, unfortunately, it also caused a collision with this "private" argument